src/new_hope/week1.clj

```diff
@@ -65,3 +65,85 @@
     (crit/quick-bench (second (reverse coll)))
     (crit/quick-bench (last (butlast coll))))
   )
+
+;; Homework
+
+;; Sum It All Up
+(defn sum [coll] (if (empty? coll) 0 (+ (first coll) (sum (rest coll)))))
```

Comment:
This solution is enough to qualify for the homework, but IRL it has issues:
1. Blows the stack when coll is longer than ~10,000 (use tail recursion instead)
2. Prone to integer overflow (use `+'`)